src/bp/ui-admin/src/Pages/Login.js

```diff
@@ -16,11 +19,17 @@ export default class Login extends Component {
   loadAuthConfig = async () => {
     const { data } = await api.getAnonymous().get('/auth/config')
 
-    this.setState({ isFirstTimeUse: data.payload.isFirstTimeUse })
+    this.setState({
```

Comment:
Not necessary for now ... This auth logic should be moved into redux actions & data should be stored/accessible in the store

Reply:
It is required to redirect to the register page if there's no other account on the workspace

Reply:
That doesn't prevent moving it in a redux action

Reply:
It could be, i'll check after it's merged with admin-ui to avoid conflict since reducers were moved